### ISSUE TITLE
[Feature Request] Save scan locations

### ISSUE THREAD

**Issue by Chris221** [favll/pogom#327]
## Checklist
 - [x] I have read the [FAQ](https://github.com/favll/pogom/blob/master/README.md#faq)
 - [x] I have checked if this issue already exists (both open and closed issues)
 - [x] I am using the most recent version of pogom from the master branch
 - [ ] If this is an installation issue, I am posting my OS, OS version, python version, pip version
 - [ ] I'm posting the complete log
 - [x] I'm using a descriptive title for this issue

## My Issue
It would be really nice as a feature to save a set of locations to scan for say night time hunting/day time hunting. Or have a bunch for the beach, parks, or around the town.

Basically have the ability to save a set of locations for quickly switching. So for example, I could remove and add a bunch of locations if I am going to play at the beach.
...

**Comment by qwerty0123** [comment]
I do this now with multiple configs, but this would be an awesome feature

**Comment by Chris221** [comment]
@qwerty0123 Yea I know you can do that and that's why I thought this would be cool idea, Otherwise I remote into my computer at my house and kill my data lol

**Comment by qwerty0123** [comment]
Agreed. It would be an awesome idea
